instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -30,12 +33,14 @@
  */
 @Configuration
 @EnableConfigurationProperties(SamplerProperties.class)
+@Import(PropagationAutoConfiguration.class)
```

Comment:
Is that annotation needed? All other configurations use `@AutoConfigureBefore` and that seems to be enough to guarantee the correct order

Reply:
You're right - I lacked the right configuration in the tests and fixing it made this line obsolete, thanks!